### PR TITLE
Update aws-efs-csi-driver-operator to use stable channel

### PR DESCRIPTION
**Pull request by dobsonj**
/hold for https://github.com/openshift/aws-efs-csi-driver-operator/pull/41 to merge first.
/cc @openshift/storage

### CODE REVIEW

**Comment by dobsonj** [comment]
operator-e2e is failing because we need https://github.com/openshift/aws-efs-csi-driver-operator/pull/41 to merge first
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/28071/rehearse-28071-pull-ci-openshift-aws-efs-csi-driver-operator-master-operator-e2e/1517552120826433536/artifacts/operator-e2e/optional-operators-subscribe/artifacts/sub-oo-98r5k.yaml
```
  - message: 'constraints not satisfiable: no operators found in channel preview of
      package aws-efs-csi-driver-operator in the catalog referenced by subscription
      oo-98r5k, subscription oo-98r5k exists'
```

**Comment by dobsonj** [comment]
/hold until I can get this working... the preview channel is still not found, I must be missing something.
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/28071/rehearse-28071-pull-ci-openshift-aws-efs-csi-driver-operator-master-operator-e2e/1518911945850228736/artifacts/operator-e2e/optional-operators-subscribe/artifacts/sub-oo-hxwmk.yaml

**Comment by dobsonj** [comment]
> /hold until I can get this working... the preview channel is still not found, I must be missing something.

I missed `config/metadata/annotations.yaml` before and filed https://github.com/openshift/aws-efs-csi-driver-operator/pull/42 to address it. Also setting everything to the stable channel instead of trying to use `preview` in CI.

File: ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-master.yaml [hunk 1]
```diff
@@ -11,18 +11,14 @@ base_images:
...
-    tag: rhel-8-golang-1.17-openshift-4.10
   ocp_builder_rhel-8-golang-1.17-openshift-4.11:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.17-openshift-4.11
   operator-index:
     name: redhat-operator-index
     namespace: ci
-    tag: v4.8
+    tag: v4.10
```
[the remark this answers is not in the document]
Comment (dobsonj) [reply ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-master.yaml:21]: I read Hemant's comment [here](https://github.com/openshift/release/pull/20632/files#r682670829) and assumed the 4.11 image would not be available yet. I grepped for all the other places where this image is used, and they're still on 4.9 or 4.8.

[the remark this answers is not in the document]
Comment (dobsonj) [reply ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-master.yaml:21]: Yeah, here we go, 4.10 was just added a couple of days ago:
https://github.com/openshift/release/blob/master/clusters/app.ci/supplemental-ci-images/redhat-operator-index.yaml#L8

**Comment by gnufied** [comment]
Do we need a follow up PR in operator repo for changing the channel to stable?

**Comment by dobsonj** [comment]
> Do we need a follow up PR in operator repo for changing the channel to stable?

It already merged in https://github.com/openshift/aws-efs-csi-driver-operator/pull/42

**Pull request merged**